src/models/DemandControl.ts

```diff
@@ -16,4 +17,46 @@ export class DemandControl {
         result.maxPower = DaikinDataParser.resolveInteger(dict, 'max_pow');
         cb(null, 'OK', result);
     }
+
+    public overwrite(obj: Partial<DemandControl>) {
+        if (obj.type !== undefined) this.type = obj.type;
+        if (obj.enabled !== undefined) this.enabled = obj.enabled;
+        if (obj.mode !== undefined) this.mode = obj.mode;
+        if (obj.maxPower !== undefined) this.maxPower = obj.maxPower;
+    }
+
+    // According to https://www.akkudoktor.net/forum/postid/190147/
+    public getRequestDict(): RequestDict {
+        if (this.enabled == undefined) throw new Error('Required Field enabled does not exist');
+
+        const dict: RequestDict = {};
+        dict['lpw'] = ''; // Appeared in all examples but doesn't seem to be necessary
+        dict['en_demand'] = this.enabled ? 1 : 0;
+        if (!this.enabled) {
+            // Switch off demand control
+            // http://<DAIKIN-IP>/aircon/set_demand_control?lpw=&en_demand=0
+            return dict;
+        }
+
+        // Enable and set demand control
+        // http://<DAIKIN-IP>/aircon/set_demand_control?lpw=&en_demand=1&mode=0&type=1&max_pow=[40-100]
+
+        // For automatic or manual demand control we need type==1, mode and maxPower
+        if (this.type !== 1) throw new Error('Required Field type != 1. Assume no support for demand control');
+        if (this.mode === undefined) throw new Error('Required Field mode do not exists');
+        if (this.maxPower === undefined) throw new Error('Required Field maxPower do not exists');
+
+        // For now we do support automatic or manual mode only
+        if (this.mode !== 0 && this.mode !== 2)
+            throw new Error('Currently only automatic(2) or manual mode(0) is supported');
+
+        // maxPower must be modulo 5 and between 40 and 100
+        if (this.maxPower < 40 || this.maxPower > 100 || this.maxPower % 5 !== 0)
```

Comment:
For convenience add rounding to round to next allowed value?

Reply:
Ok. Good idea.

Reply:
Change committed. Let me give it a try in my iobroker instance first.

Reply:
Looks good. Rounding works as expected.

Reply:
Thanks, will review tomorrow